### PR TITLE
Brakeman: security checker

### PR DESCRIPTION
**Pull request by mattscilipoti**
Adds brakeman to /metrics.
- outputs to "public/metrics/brakeman"
- https://github.com/presidentbeef/brakeman

To test manually, use rake tasks: `rake -T metrics`

Note: immediately after pushing, you will receive "No route matches [GET] "/metrics/brakeman" because the report as not been generated (public/metrics/brakeman/index.html).  You can either generate the report, via the rake task, or wait for the nightly cron/rake task.

### CODE REVIEW

File: app/models/metric.rb [hunk 1]
```diff
@@ -0,0 +1,7 @@
+require 'logger'
```
Comment (adambray) [inline app/models/metric.rb]: Is this required?

**Comment by adambray** [comment]
👍

File: app/models/metric.rb (hunk 1 above)
Comment (mattscilipoti) [inline app/models/metric.rb]: Not any more.  Removing.

**Comment by nolds9** [comment]
needs team review

**Comment by RobertAKARobin** [comment]
We don't know what this does. Can re-open later if we find a need.

**Pull request closed without merging**